screens/HomePage.tsx

```diff
@@ -21,7 +22,7 @@ const HomePage = () => {
       </TouchableOpacity>
       </View>
       <ScrollView style={styles.scrollView}>
-        <ItemCard />
+        <ItemCard style={styles.cardStyle} />
         <ItemCard />
         <ItemCard />
         <ItemCard />
```

Comment:
`ScrollView` is a bit off. You can see that the scrolling bar is not stuck to the right side of the screen, but rather to the container. This is probably a styling issue with margins.